src/shared/bus-util.h

```diff
@@ -23,11 +23,31 @@ typedef enum BusTransport {
 } BusTransport;
 
 typedef struct BusLocator {
-        const char    *destination;
-        const char    *path;
-        const char    *interface;
+        const char *destination;
+        const char *path;
+        const char *interface;
 } BusLocator;
 
+typedef struct BusObjectImplementation BusObjectImplementation;
+
+typedef struct BusObjectVtablePair {
+        const sd_bus_vtable *vtable;
+        sd_bus_object_find_t object_find;
+} BusObjectVtablePair;
+
+struct BusObjectImplementation {
+        const char *path;
+        const char *interface;
+        const BusObjectVtablePair *vtables;
+        sd_bus_node_enumerator_t node_enumerator;
+        bool manager;
```

Comment:
not used?

Reply:
It's for systemd-homed. I didn't convert it yet, but it's on my todo list (along with the api docs for it...).

Reply:
add the bool then back in?